core/src/main/java/nu/yona/server/subscriptions/entities/BuddyConnectResponseMessage.java

```diff
@@ -15,11 +14,6 @@
 @Entity
 public class BuddyConnectResponseMessage extends BuddyConnectMessage
 {
-
-	@Transient
-	private UUID destinationID;
```

Comment:
I thought that these columns accidentally had the same name. This column was used nowhere!